reports/Google-2018-report.md

```diff
@@ -54,9 +55,14 @@ As a result, the number of unique visitors grew from less than 5 to peaks of ove
 
 :o: **Problem** International project management and communication
 
-:heavy_check_mark: **Solution** The members and collaborators of this project were located in Canada, Britain, the Netherlands, the USA, and India. The strategy for this problem was to work publicly. This meant rather than waiting until completion to share something, we would continuously post our work in progress and incorporate feedback. Specifically, Github milestones and “closing soon” tags were helpful to highlight weekly tasks and mark them for review. 
+:heavy_check_mark: **Solution** The members and collaborators of this project were located in Canada ([Elizabeth DuPre](https://github.com/emdupre)), Britain ([Kirstie Whitaker](https://github.com/KirstieJane)), the Netherlands ([Dora Hermes](https://github.com/dorahermes)), the USA ([Chris Gorgolewski](https://github.com/chrisfilo) and [Chris Holdgraf](https://github.com/choldgraf/)), and India **[PATRICK - who was in India?? I feel terrible that I don't know!]**.
```

Comment:
Add in link to all collaborators.

Reply:
I mentioned Madhur Tandon for his help (providing the questions) in forming the BIDS App FAQ